assets/_scss/elements/_typography.scss

```diff
@@ -175,3 +177,15 @@ dfn {
 .usa-drop_text {
   margin-bottom: 0;
 }
+
+.usa-background-dark {
+  background-color: #333B46;
+
+  p {
```

Comment:
@msecret I added span in aea5292. If you let me know which form elements you think should be added, then I can add it as well.